projects/extension/src/containers/Options.tsx

```diff
@@ -175,8 +184,10 @@ const Options: React.FunctionComponent = () => {
           {/** Networks section */}
           <TabsContent activeTab={activeTab}>
             <section>
-              {bg?.uiInterface?.smoldotCrashError ? (
-                <ClientError error={bg?.uiInterface?.smoldotCrashError} />
+              {clientError || bg?.uiInterface.smoldotCrashError ? (
+                <ClientError
+                  error={clientError || bg?.uiInterface.smoldotCrashError || ""}
+                />
               ) : networks.length ? (
```

Comment:
I also don't think it makes sense to hide the list of chains if there is an error.
Again, you're assuming that "client error" is equal to "no chain", but I don't see a reason to put this assumption into the code.

Reply:
Im not sure why you call an "assumption" the case when the Smoldot panics (aka crashes). AFAI there are no networks running on smoldot light client when it crashes.

Reply:
My point is that it is not _necessarily_ correct.
The UI should not make any assumption about what happens in the background. If the background says that there is a crash error then the UI should display the crash error, if the background says there are networks then the UI should display the networks, and if the background says there are both then the UI should display both. It's as simple as that.